<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IntroMovieSwapper</class>
 <widget class="QDialog" name="IntroMovieSwapper">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>611</width>
    <height>261</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>611</width>
    <height>261</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>611</width>
    <height>261</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Intro Movie Swapper</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../data/icon.png</normaloff>../data/icon.png</iconset>
  </property>
  <property name="toolTipDuration">
   <number>0</number>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QGroupBox" name="dirGroup">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>591</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>폴더 선택</string>
   </property>
   <widget class="QLineEdit" name="folderDir">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>491</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="dragEnabled">
     <bool>true</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>경로를 지정해주세요</string>
    </property>
   </widget>
   <widget class="QPushButton" name="selFolder">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>20</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>폴더 선택</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="seeDirGroup">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>69</y>
     <width>501</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>파일 선택</string>
   </property>
   <widget class="QListWidget" name="dirInfo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>481</width>
      <height>101</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>80</y>
     <width>85</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>8</number>
    </property>
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <item>
     <widget class="QPushButton" name="fileSwap">
      <property name="text">
       <string>교  체</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="fileRestore">
      <property name="text">
       <string>복  원</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="shutDown">
      <property name="text">
       <string>종  료</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>541</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="textVisible">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="Version">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>242</y>
     <width>56</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string>ver 0.1</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
   <property name="margin">
    <number>1</number>
   </property>
   <property name="indent">
    <number>2</number>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::NoTextInteraction</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
